client/scss/components/_button.scss

```diff
@@ -164,7 +164,10 @@
 
       .icon {
         @include svg-icon(1rem);
-        padding: 0.75em;
+        padding: 0.36em;
+        @media screen and (min-width: $screen-sm-min) {
+          padding: 0.75em;
+        }
```

Comment:
is there a way we can solve this without adding a new screen width?

not sure as I have not run the code myself, but maybe the line-height approach from https://css-tricks.com/centering-css-complete-guide/

or flexbox?